frappe/tests/test_commands.py

```diff
@@ -474,6 +474,9 @@ def test_make_app(self):
 		# cleanup
 		shutil.rmtree(test_app_path)
 
+	def test_drop_site(self):
+		self.execute("bench new-site test_site.localhost --maria-db-password {frappe.conf.db_password}")
+		self.execute("bench drop-site test_site.localhost --root-password {frappe.conf.db_password}")
```

Comment:
We'll have to check for:
* Success return code
* Database actually being dropped
* Archived site exists in the archived folder

at the very least for drop-site tests I think